recurring_contract/migrations/14.0.1.0.5/post-migration.py

```diff
@@ -0,0 +1,47 @@
+import logging
+
+from dateutil.utils import today
+from openupgradelib import openupgrade
+
+_logger = logging.getLogger(__name__)
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    # Retrieve balance product
+    balance_product = env["product.product"].search([("name", "=", "Balance")])
+    if len(balance_product.ids) != 1:
+        _logger.warning("balance product is not unique")
+        return 1
+    # retrieve all contract line
+    contract_lines = env["recurring.contract"].search([("state", "in", ["waiting", "active"])]).mapped(
```

Comment:
What about "draft" ? I think they should be included.

Reply:
The draft state is currently not used, when we call the generation of invoices the contract is in waiting state,

Reply:
I think here also including draft is needed.